crates/holochain/tests/websocket/mod.rs

```diff
@@ -397,9 +398,9 @@ async fn emit_signals() {
     assert_matches!(response, AdminResponse::AppEnabled { .. });
 
     // Generate signing key pair
-    let mut rng = rand_dalek::thread_rng();
```

Comment:
With the latest version of `ed25519_dalek` we don't need a pinned `rand`, it works with the current version